docs/.vuepress/containers/conditionalBlock/conditionalBlock.js

```diff
@@ -0,0 +1,43 @@
+const { getBuildDocsFramework } = require('../../helpers');
+
+/* eslint-disable jsdoc/require-description-complete-sentence */
+/**
+ * Container used to display/hide blocks of content relevant to specific frameworks.
+ * It relies on providing the framework name(s) as arguments to the container (see 'Usage' below) and setting a
+ * `FRAMEWORK` environmental variable as the framework name, for example:
+ *
+ * ```
+ * FRAMEWORK=react
+ * ```
+ *
+ * Usage:
+ * ```
+ * ::: only-for react
+ * Content to be displayed only for React documentation.
+ * :::
+ *
+ * ::: only-for javascript react vue
+ * Content to be displayed only for JS, React and Vue documentation.
+ * :::
+ * ```
+ */
+module.exports = {
+  type: 'only-for',
+  render(tokens, index) {
+    const framework = getBuildDocsFramework() || 'javascript';
```

Comment:
The `getBuildDocsFramework` won't work for `npm run docs:start` script, so sorry for this suggestion. There should be used something else. Maybe `parseFramework` helper will be okey.

https://github.com/handsontable/handsontable/blob/3e77e0bc759b2c2aaa20b2a3ef68daaac13ef51f/docs/.vuepress/helpers.js#L156-L162

Please change it.